src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -142,17 +142,17 @@ class GrowableArrayView : public GrowableArrayBase {
   }
 
   E& at(int i) {
-    assert(0 <= i && i < _len, "illegal index");
+    assert(0 <= i && i < _len, "illegal index %d, %d accessible elements", i, _len);
```

Comment:
How about:

"Illegal index %d for length %d"

?

Reply:
That sounds like a good suggestions !